filters/include/pcl/filters/impl/normal_space.hpp

```diff
@@ -59,11 +59,10 @@ pcl::NormalSpaceSampling<PointT, NormalT>::initCompute ()
     return false;
   }
 
-  boost::mt19937 rng (static_cast<unsigned int> (seed_));
-  boost::uniform_int<unsigned int> uniform_distrib (0, unsigned (input_->size ()));
-  delete rng_uniform_distribution_;
-  rng_uniform_distribution_ = new boost::variate_generator<boost::mt19937, boost::uniform_int<unsigned int> > (rng, uniform_distrib);
-
+  rng_uniform_distribution_ = std::bind (
```

Comment:
`std::bind` is [frowned upon in C++14](https://stackoverflow.com/a/17545183). Lambdas are preferred